core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.spec.ts

```diff
@@ -108,4 +117,24 @@ describe('TemplateBuilderComponent', () => {
         spectator.component.deleteRow('123');
         expect(removeRowMock).toHaveBeenCalledWith('123');
     });
+
+    it('should call editRowStyleClasses from store when clicking on row-style-class-button', () => {
+        const editRowStyleClassesMock = jest.spyOn(spectator.component, 'editRowStyleClasses');
```

Comment:
Don't mock the methods component you're testing... you can mock the external things this method is calling tho.